### DISCUSSION TITLE
有设置定时执行某insert sql的功能吗

### DISCUSSION THREAD

**Discussion by cdavis26424** [apache/doris#8742] [A - General / Q&A] [answered]
有一个类似如下sql的语句，需要定时执行
INSERT INTO test (c1, c2) SELECT * from test2;
有设置定时执行某sql的功能吗

版本0.15

**Comment by dataalive** [comment, marked as answer]
可能需要借助于 dophine scheduler 的调度工具，目前doris自己没有配置调度工具。
